### DISCUSSION TITLE
How to stop pulsar pod?

### DISCUSSION THREAD

**Discussion by LiuAlex1109** [milvus-io/milvus#32735] [Q&A and General discussion]
I am a beginner for using k8s，and I have installed Milvus using the Kubernetes (k8s) cluster as described in the official documentation. How can I delete the Pulsar-related pods and make them not restart? pods are list below:
```
milvus-minio-79686c7bc-w7j57                 1/1     Running     0             24h
milvus-pulsar-bookie-0                       1/1     Running     0             24h
milvus-pulsar-bookie-1                       1/1     Running     0             24h
milvus-pulsar-bookie-init-5zclh              0/1     Completed   0             24h
milvus-pulsar-broker-0                       1/1     Running     0             24h
milvus-pulsar-proxy-0                        1/1     Running     0             24h
milvus-pulsar-pulsar-init-nbxq9              0/1     Completed   0             24h
milvus-pulsar-zookeeper-0                    1/1     Running     0             24h
```

**Comment by LoveEachDay** [comment]
@LiuAlex1109 Have you install milvus cluster using milvus helm chart or milvus operator? 
If using milvus-helm:
1. List all the pulsar component: `kubectl get sts | grep pulsar`
2. Scale pulsar component to 0: `kubectl scale sts <pulsar-sts> --replicas=0`

If using milvus operator:
1. List helm history: `helm ls`
2. Uninstall the pulsar helm deployment: `helm uninstall <pulsart-helm-deploy>`
